src/examples/src/widgets/tree/Basic.tsx

```diff
@@ -0,0 +1,36 @@
+import { create, tsx } from '@dojo/framework/core/vdom';
+import Tree from '@dojo/widgets/tree';
+
+const factory = create();
+
+export default factory(function Basic() {
+	const nodes = [
+		{
+			content: 'parent 1',
+			expanded: true,
```

Comment:
I think that things like `expanded` / `selected` etc should be moved out of the data, perhaps array of ids allowing for both controlled / partially controlled scenario?

Reply:
Implemented as `expandedNodes?: string[]` and `selectedNotes?: string[]`